contracts/V1/StateChannelDiamondProxy/LocalDiamond.sol

```diff
@@ -287,4 +291,32 @@ contract LocalDiamond is StateChannelManagerProxy {
             dispute, disputeAuditingData, auditingDataIntegrityVerified
         );
     }
+
+    function isDisputeWindowRelevant(bytes32 channelId, bytes32 disputedForkId) external view returns (bool) {
+        StateSnapshot storage currentStateSnapshot = stateSnapshots[channelId];
+        DisputeData storage disputeData = disputeData[channelId];
+
+        bytes32 currentForkId = currentStateSnapshot.forkId;
+
+        if (currentForkId == disputedForkId) {
+            return true;
+        }
+
+        DisputeWindow storage disputeWindow = disputeData.disputeWindowMap[currentForkId];
+
+        while (
+            disputeWindow.reducedResult.forkId != bytes32(0)
+                && _isReduceChallengePeriodExpired(disputeWindow, getEvidenceTime())
+        ) {
+            currentForkId = disputeWindow.reducedResult.forkId;
+
+            if (currentForkId == disputedForkId) {
+                return true;
+            }
+
+            disputeWindow = disputeData.disputeWindowMap[currentForkId];
+        }
+
```

Comment:
A dispute is relevant if it can change our fork choice by altering a disputeWindow so it reduces to something else - already reduced and finalized dispute windows can't change - so we have to find the ones along the path that are not finalized and observe them

Reply:
removed